test/syntax.jl

```diff
@@ -3060,3 +3060,21 @@ end
     @test err == 5 + 6
     @test x == 1
 end
+
+@test_throws ParseError Meta.parse("""
+function checkUserAccess(u::User)
+	if u.accessLevel != "user\u202e \u2066# users are not allowed\u2069\u2066"
```

Comment:
Any reason this was indented with tabs?

Reply:
That may be from the initial report/gist I sent - I just checked my local version and it seems to have a tab character there as well, though that wasn't intentional. Not sure how it got there. It certainly isn't a functional part of the vulnerability though.